python/lsst/pipe/base/configOverrides.py

```diff
@@ -234,7 +237,8 @@ def applyTo(self, config):
 
         for otype, override in self._overrides:
             if otype is OverrideTypes.File:
-                config.load(override)
+                with override.open("r") as buffer:
+                    config.loadFromStream(buffer, filename=str(override))
```

Comment:
It turns out that this breaks fgcmcal because fgcmcal has a config that is using os.path to find the enclosing directory and then joins it to a new file that is passed to `config.load()` -- this is now a `file://` URI string and so fails. The real fix is to (1) make `config.load` support `ResourcePath` and (2) consider what we are going to do with all the configs that assume local files when they could actually be s3 URIs. Obviously treating an s3 URI like a file will work to a certain extent.

Reply:
We may want to introduce a `eups://` or `rubinsrc://` scheme or something to `ResourcePath`, as a way to let configs refer to config files that are part of the currently-setup LSST Science Pipelines software version.  >99% of any path manipulations inside config files are to compute things relative to the result of a call to `utils.getPackageDir`.

Reply:
`eups://` is fine with me -- resource path does lazy loading of the per-scheme plugins. The other bigger option is to put the configs into the python packages properly so that they look like `pkg_resource` resources.

Reply:
I'm not a big fan of how pkg_resources forces everything to be in the Python package dir; having configs in a separate directory in the git repo is a lot nicer.  I'd be willing to give it up if it's just one of a few things getting us out of EUPS and into something more consistent with the way the rest of the world does Python development, but we're not particularly close to that right now.